lib/node_modules/@stdlib/utils/dsv/README.md

```diff
@@ -57,6 +57,7 @@ The namespace contains the following utilities:
 
 <div class="namespace-toc">
 
+
```

Comment:
@Planeshifter This is another instance where it would be good to make the Markdown processing of namespace TOCs and README related packages consistent. This gap was removed in a prior PR (see https://github.com/stdlib-js/stdlib/pull/1248/files) and its reintroduction is just creating unnecessary PR and commit noise.

Reply:
@kgryte Hopefully, this should now be resolved by https://github.com/stdlib-js/stdlib/commit/bf71cdee618bd07cd74dde910fc8c9503e467fae.